### ISSUE TITLE
Unity Web Player & Shockwave ?

### ISSUE THREAD

**Issue by unblocked-games-school** [ruffle-rs/ruffle#6729]
Hello,

Does Ruffle work for games developed with unity web player and shockwave?

**Comment by Herschel** [comment]
Hi, Ruffle only works with Adobe Flash files (SWF file format). Director, Shockwave, or Unity Web Players are out of scope of the project (we have our hands full only with SWF!).

You may be interested in the [Flashpoint](https://bluemaxima.org/flashpoint/) project, which seeks to archive all web games including Unity and Shockwave. Thank you!
